.github/workflows/mkn.yml

```
@@ -0,0 +1,39 @@
+    - uses: actions/checkout@v4
+
+    - name: Install system deps (openmpi, hdf5, ccache,...)
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libopenmpi-dev openmpi-bin libhdf5-openmpi-dev ccache plocate
+        sudo updatedb
+
+    - name: "setup"
+      env:
+        MKN_GCC_PREFERRED: 1
+      run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:3:34: Double quote to prevent globbing and word splitting

(shellcheck)